package.json

```diff
@@ -8,13 +8,13 @@
     "test": "stylelint test/*.css && stylelint test/*.scss --config sass.js"
   },
   "dependencies": {
-    "stylelint-config-standard": "^34.0.0",
+    "stylelint-config-standard": "^36.0.1",
     "stylelint-order": "^6.0.4"
   },
   "peerDependencies": {
-    "postcss": "^8.4.5",
-    "postcss-scss": "^4.0.2",
-    "stylelint": "^15.11.0"
+    "postcss": "^8.4.47",
+    "postcss-scss": "^4.0.9",
```

Comment:
technically these don't have to change, right? since for peer dependencies we probably want the widest possible range

Reply:
Hmm not sure. At least for `postcss`, I set the version that `stylelint` indicates in their dependencies: https://github.com/stylelint/stylelint/blob/afbab11aacde5b0f59e3c755e2bd5fd4e5e320da/package.json#L157

Reply:
@gluxon what are your thoughts on this?

Reply:
sorry forgot to reply - if that's the version stylelint needs than this makes sense!